src/tests/routes/test_groups.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import json
+import pytest
+from datetime import datetime
+
+from app import db
+from app.api import crud
+from tests.db_utils import get_entry, fill_table
+from tests.utils import update_only_datetime
+
+GROUP_TABLE = [
+    {"id": 1, "name": "first_group", "created_at": "2020-10-13T08:18:45.447773"},
+    {"id": 2, "name": "second_group", "created_at": "2020-09-13T08:18:45.447773"}
+]
+
+
+ACCESS_TABLE = [
+    {"id": 1, "login": "first_login", "hashed_password": "hashed_pwd", "scopes": "user"},
+    {"id": 2, "login": "second_login", "hashed_password": "hashed_pwd", "scopes": "admin"},
+    {"id": 3, "login": "third_login", "hashed_password": "hashed_pwd", "scopes": "device"},
+]
+
+
+def compare_entries(ref, test):
+    for k, v in ref.items():
+        if isinstance(v, float):
+            # For float issues
+            assert abs(v - test[k]) < 1E-5
+        else:
+            assert v == test[k]
```

Comment:
Considering that there is no float field in the group table, perhaps we don't need this?